libbroker/broker/internal/peering.hh

```diff
@@ -38,7 +38,7 @@ public:
 
   /// Forces the peering to shut down its connection without performing the BYE
   /// handshake.
-  void force_disconnect();
+  void force_disconnect(const std::string& reason = "");
```

Comment:
```suggestion
  void force_disconnect(std::string reason);
```

Not a fan of the empty default. There are two more places where we call `force_disconnect` that would currently use the default:
- `core_actor_state::finalize_shutdown`: can pass something like "shutting down"
- `peering::schedule_bye_timeout`: can pass something like "timeout during graceful disconnect"

I think differentiating those with dedicated reasons would make sense as well.

Reply:
Yeah I was just trying to keep this minimal. I like your suggestions, will do. Thanks!